### ISSUE TITLE
AREPL doesnt work after 2.0 update

### ISSUE THREAD

**Issue by w1nsid** [Almenon/AREPL-vscode#363]
Description 
AREPL fails to launch after latest update. Devtools outputs following error message :

`
notificationsAlerts.ts:38 Error: Running the contributed command: 'extension.currentAREPLSession' failed.
	at _executeContributedCommand (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:699:849)
	at _.$executeContributedCommand (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:700:104)
	at g._doInvokeHandler (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:876:926)
	at g._invokeHandler (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:876:618)
	at g._receiveRequest (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:875:252)
	at g._receiveOneMessage (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:874:6)
	at c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:872:136
	at l.fire (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:46:67)
	at v.fire (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:256:601)
	at c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:1066:179
	at l.fire (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:46:67)
	at v.fire (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:256:601)
	at t.PersistentProtocol._receiveMessage (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:261:629)
	at c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:258:708
	at l.fire (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:46:67)
	at p.acceptChunk (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:253:989)
	at c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:253:341
	at Socket.t (c:\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:263:276)
	at Socket.emit (events.js:223:5)
	at addChunk (_stream_readable.js:309:12)
	at readableAddChunk (_stream_readable.js:290:11)
	at Socket.Readable.push (_stream_readable.js:224:10)
	at Pipe.onStreamRead (internal/stream_base_commons.js:181:23)
onDidChangeNotification @ breadcrumbsModel.ts:39
(anonymous) @ breadcrumbsModel.ts:39
fire @ event.ts:584
addNotification @ notifications.ts:204
error @ breakpointWidget.ts:390
(anonymous) @ titleControl.ts:135
fire @ event.ts:584
run @ actions.ts:135
async function (async)
run @ actions.ts:135
run @ editor.ts:1172
onClick @ menuEntryActionViewItem.ts:74
(anonymous) @ actionViewItems.ts:121
processImmediate @ internal/timers.js:439
`

I tried to downgrade to version 1.0.24 and it works as expected.

**Comment by caleb15** [comment]
Dang! What vscode and python version do you have? (You can check the arepl.pythonpath and python.pythonpath settings)

**Comment by caleb15** [comment]
Also what method are you using to launch arepl?

**Comment by w1nsid** [comment]
VS code version 👍 

```
Version: 1.51.1 (user setup)
Commit: e5a624b788d92b8d34d1392e4c4d9789406efe8f
Date: 2020-11-10T23:34:32.027Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19042
```
Python version : 3.7.9 64 bit (conda environement)

Both arepl.pythonpath and python.pythonpath settings are set to their default values , i should also mention im using the Python extension from microsoft with conda to manage python envs.

I tried to launch AREPL from the icon on the upper right corner and the command palette aswell with the same result (failed launch). As i mentioned in the issue, reverting to version 1.0.24 through the extension manager fixed the problem.
I did not hower attempt to launch it from diffrent python environements.

**Comment by Almenon** [comment]
commented under wrong account -_-

My work is starting soon but I'll download python 3.7 and get back to you tonight

**Comment by w1nsid** [comment]
Your efforts are really appreciated , thank you !

**Comment by Almenon** [comment]
I tried with python 3.7.9 and I was not able to reproduce the error. This is super weird cause my setup is almost the exact same as yours:

```
Version: 1.51.1 (system setup)
Commit: e5a624b788d92b8d34d1392e4c4d9789406efe8f
Date: 2020-11-10T23:34:32.027Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19041
```

Normally I would say to check the devtools output but you already did that and the error unfortunately doesn't make it clear what the issue is. [Extension.ts](https://github.com/Almenon/AREPL-vscode/blob/master/src/extension.ts) hasn't changed since the last update... :(

Maybe try deleting any leftover arepl folders from `C:\\Users\\yourUserNameHere\\.vscode\\extensions` and reinstalling arepl v2 just in case it is some oddball glitch that doesn't happen the second time around?

Does opening arepl in a new doc (control-shift-q) work?

**Comment by w1nsid** [comment]
- Opening arepl in a new doc (control-shift-q) works well.
- If i save the newly opened doc it fails again.
- Creating a new python file or opening from an existing one does not work , also importing dump leads to an import resolve error ( module not found ).
- When using the ctrl shift q without saving , there is no import error when importing arepl_dump and everything is functional untill i save the file.
- I deleted the extension folder from the mentioned folder and it did not work.
- I tried installing on a diffrent system (mac) with the same conda setup and got the same error.


It seems like the problem only occurs with saved python scripts .

**Comment by w1nsid** [comment]
I decided to do a clean install of VS code and it fixed the problem .

**Comment by caleb15** [comment]
Woah, bizarre. Glad it's fixed!

**Comment by w1nsid** [comment]
I think i identified the trigger for this issue , it only happens when i set value of `**"AREPL.skipLandingPage"**` to true .
I hope this helps !

**Comment by Almenon** [comment]
I'm able to reproduce now, thanks 😬

**Comment by Almenon** [comment]
@w1nsid I just released a new version of AREPL, v2.0.1, does it work now?

**Comment by w1nsid** [comment]
hello , thanks for the update i confirm its functional on my system.
